python/larpix.py

```diff
@@ -16,43 +16,155 @@ def __init__(self, chip_id, io_chain):
         self.chip_id = chip_id
         self.io_chain = io_chain
         self.data_to_send = []
+        
+        self.configuration = Configuration()
 
     def set_pixel_trim_thresholds(self, thresholds):
-        pass
+        if len(thresholds) != l.larpix_num_channels():
+            return 1
+        for i,value in enumerate(thresholds):
+            self.configuration.pixel_trim_thresholds[i] = value
+        return 0
 
     def set_global_threshold(self, threshold):
-        pass
+        self.configuration.global_threshold = threshold
+        return 0
 
     def set_csa_gain(self, gain):
-        pass
-
-    def enable_
+        self.configuration.csa_gain = gain
+        return 0
+
+    def enable_channels(self, map):
+        self.configuration.channel_mask = self.configuration.channel_mask & ~map 
```

Comment:
Here `self.configuration.channel_mask` is a c array (defined [here](https://github.com/samkohn/larpix-control/pull/5/files#diff-bf8d555f276f235c897b5bbe727efc42R46)) so this assignment doesn't work.